CHANGES.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- [NEW] Add Detect Secrets baseline file to scan changes for secrets in code.
```

Comment:
We can drop this CHANGES entry it is not relevant for users.

Reply:
Removed it in 7e3d03b